README.md

```diff
@@ -9,7 +9,55 @@ The main folder contains the code necessary to generate these client libraries.
 **NOTE: These generated clients are under development and should be considered
 experimental!**
 
-## Setup
+
+## Usage
+
+All available Google API clients can be found [on hex.pm][hex_pm]. Add a client
+to your project's `mix.exs` under `deps`:
+
+```ex
+defmodule YourApplication.Mixfile do
+  use Mix.Project
+  #...
+
+  # Run "mix help deps" to learn about dependencies.
+  defp deps do
+    [
+      {:google_api_storage, "~> 0.0.1"},
+      {:goth, "~> 0.6.0"}
```

Comment:
We could make `goth` a real dependency.

If we do, we could make a version of each API call that doesn't require a `Connection` struct as the first option, and implicitly fetches a token with necessary scopes.

The reason that the first iteration did not do this is the notion of "explicit > implicit" in Elixir.

Reply:
I think explicit seems to be more in line with the elixir paradigm. But it seems odd to not at least require goth as a dependency of `google_api_xyz`